src/web/client/utilities/commonUtil.ts

```diff
@@ -223,3 +224,43 @@ export function getBackToStudioURL() {
         .replace("{.region}", region.toLowerCase() === STUDIO_PROD_REGION ? "" : `.${WebExtensionContext.urlParametersMap.get(queryParameters.REGION) as string}`)
         .replace("{webSiteId}", WebExtensionContext.urlParametersMap.get(queryParameters.WEBSITE_ID) as string);
 }
+export function getSupportedImageFileExtensionsForEdit() {
+    return ['png', 'jpg', 'webp', 'bmp', 'tga', 'ico', 'jpeg']; // Luna paint supported image file extensions
+}
```

Comment:
Why do you need an allow list for this? There are plenty of other files that may also be binary files that, for example the hex editor supports opening _any_ file extension to view it's data as hex?

Reply:
This allow list is just for write back to our backend - we allow reading of a file using any editor that vscode supports. Following up more on chat around this.

Reply:
updating list per Luna paint supported extensions list.